Geometry/CSCGeometry/src/CSCLayerGeometry.cc

```diff
@@ -49,11 +52,11 @@ CSCLayerGeometry::CSCLayerGeometry( const CSCGeometry* geom, int iChamberType,
 
   if ( ! geom->realWireGeometry() ) {
     // Approximate ORCA_8_8_0 and earlier calculated geometry...
-    float wangler = wireAngleInDegrees*degree; // convert angle to radians
+    float wangler = wireAngleInDegrees * 1.0_deg; // convert angle to radians
     float wireCos = cos(wangler);
     float wireSin = sin(wangler);
     float y2 = apothem * wireCos + hBottomEdge * fabs(wireSin);
-    float wireSpacing = wg.wireSpacing/10.; // in cm
+    float wireSpacing = CONVERT_TO( wg.wireSpacing, cm );
```

Comment:
are we using Apache/Drill?
https://drill.apache.org/docs/data-type-conversion/#convert_to-and-convert_from

there is also a convert_to (function) in boost...

I would strongly recommend to add say a "DD_" leader at least

anyhow excellent move just after releasing the new cams coding conventions....

Reply:
That's a good point about the possible name clash. In my next PR, I'll change `CONVERT_TO` to `CONVERT_UNITS_TO` or maybe `CMS_CONVERT_UNITS` in `DDUnits.h` and all other files. Eventually, code in Alignment and Simulation will be migrated in the same way, so the naming will need to be generic CMSSW and not specific to Detector Description.